chore: undeprecating retirement

deborahgu opened the pull request:
without  doing any Python or Ubuntu upgrades:

- Remove deprecation markers
- remove demographics

APER-4019

feanil commented:
:+1:

doctoryes commented on scripts/format_rsa_key.py:
```diff
@@ -0,0 +1,34 @@
+from os import path
+
+# Add top-level module path to sys.path before importing tubular code.
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
```
You don't need these lines if you're not importing any tubular code.

deborahgu committed:
chore: undeprecating retirement

without  doing any Python or Ubuntu upgrades:

- Remove deprecation markers
- remove demographics

justinhynes commented on README.md:
```diff
@@ -29,11 +31,9 @@ and migrated [here](https://github.com/openedx/edx-platform/tree/master/scripts/
-The migration process was completed through this Pull Request: [PR #34063](https://github.com/openedx/edx-platform/pull/34063)
-
-**2U Fork:** After being deprecated for Open edX, this repo has been forked into the `edx` GitHub org and is maintained by (and for) 2U.
+After being deprecated for Open edX, this repo was forked into the `edx` GitHub org and is maintained by (and for) 2U. Initially, on 13 March, 2024 2U switched from `tubular` to `edx-platform` versions of the retirement scripts, but in May 2025, 2U  temporarily switched back to running user from this repository, in order to most efficiently solve the problem of divergent third-party retirement st ...
```
NIt: are we missing a word? 
```... but in May 2025, 2U  temporarily switched back to running user from this repository, in order to most efficiently solve the problem of divergent third-party retirement streams.```
should it be user retirement?

justinhynes reviewed (approved):
LGTM, with one nit on a sentence in the README.

deborahgu committed:
chore: readme fixes

typos
